dist/configure

```diff
@@ -44,6 +44,10 @@ detect_ocaml () {
 }
 
 check_no_bug81300 () {
+  if [[ "$cross_build" == "1" ]]; then
+    # We don't want to build anything with march=native
+    return 0
+  fi
```

Comment:
I guess we never cross-compile for x64 so that's ok?

Reply:
Even if we'd be cross-compiling for x64 (like on an ARM-based Mac), we wouldn't want to use this because it uses `-march=native`. I guess no one is doing this (yet). But maybe we should improve this check to work for cross-compilation.